config/webpack.dev.config.js

```diff
@@ -8,21 +8,43 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
 
 module.exports = Merge.smart(commonConfig, {
   devtool: 'cheap-module-eval-source-map',
-  entry: [
-    `${require.resolve('webpack-dev-server/client')}?/`,
-    require.resolve('webpack/hot/dev-server'),
-    require.resolve('react-error-overlay'),
-  ],
```

Comment:
I'm able to remove this because the `--hot --inline` options to the webpack-dev-server does this for us automatically.